delivery_dhl_parcel/models/delivery_carrier.py

```diff
@@ -89,13 +90,17 @@ def _prepare_dhl_parcel_shipping(self, picking):
         :returns dict values for the connector
         """
         self.ensure_one()
+        # Minimum 1kg
```

Comment:
Can you add some reasoning why is this needed?

Reply:
The minimum weight for a DHL delivery is 1kg, it fails with anything under

Reply:
Then please add such comment in the code instead of a barely "Minimum 1kg", and even better if you include the returned error by DHL in this comment.

Reply:
I replaced it with the actual error response from DHL. Full response in commit message